server/routes/jobs.js

```diff
@@ -0,0 +1,61 @@
+let express = require('express');
+let Queue = require('bull');
+const { resolve } = require("path");
+let REDIS_URL = 'redis://127.0.0.1:6379';
+
+//const client = require('../')
+
+let router = express();
+//create a queue and connect with redis
+let workQueue = new Queue('work', REDIS_URL);
+
+/*app.get('/bull', function(req, res){
+  console.log("hi");
+  console.log({ root: join(__dirname, '../client/public')});
+  res.sendFile('indexBull.html', { root: join(__dirname, '../client/public') });
+});*/
+
+router.get('/', function(req, res){
```

Comment:
All unnecessary routes can be removed for now, we really only need the means to enqueue a task.